tests/unit/commands/test_expand_extensions.py

```diff
@@ -54,23 +57,33 @@ def test_command(new_dir, emitter):
         dedent(
             """\
         name: test-name
+        base: core22
+        build-base: core22
+        compression: xz
         version: '0.1'
         summary: testing extensions
         description: expand a fake extension
-        base: core22
+        confinement: strict
+        grade: stable
+        architectures: []
+        assumes: []
```

Comment:
maybe validate but don't use the validated data or ensure that these empty values don't get expanded